cpp/src/io/parquet/reader_impl.cu

```diff
@@ -185,15 +185,19 @@ type_id to_type_id(SchemaElement const& schema,
  */
 constexpr int32_t to_clockrate(type_id timestamp_type_id)
```

Comment:
This function is totally redundant with the other one.

Reply:
Moved this function to a new `io/utilities/time_utils.cuh` header.